lib/lndclient/LndClient.ts

```diff
@@ -209,10 +215,14 @@ class LndClient extends SwapClient {
 
           /** The new lnd pub key value if different from the one we had previously. */
           let newPubKey: string | undefined;
+          let newUris: string[] | undefined;
           if (this.identityPubKey !== getInfoResponse.getIdentityPubkey()) {
             newPubKey = getInfoResponse.getIdentityPubkey();
             this.logger.debug(`pubkey is ${newPubKey}`);
             this.identityPubKey = newPubKey;
+            newUris = getInfoResponse.getUrisList();
```

Comment:
Right now we'll only update `newUris` if the identity pub key changes. Maybe we should have a separate if statement that checks if the new uris list is different from the one we currently have and set `newUris` if so.

Reply:
My thinking was that `newUris` don't change at runtime so there's no need to add logic to compare array contents of `newUris`.

Reply:
But they would change if lnd stops and starts while xud is running, right? In that case I'm thinking we'd want to update our lnd uris within xud.

Reply:
Would it be OK if I create an issue for it and deal with in a separate PR?

Reply:
OK with me.